scripts/commands/updateApiDocs.ts

```diff
@@ -323,6 +343,21 @@ async function convertHtmlToMarkdown(
     JSON.stringify(toc, null, 2) + "\n",
   );
 
+  if (pkg.hasSeparateReleaseNotes) {
```

Comment:
There's now a lot of release notes logic in this file and the file is already pretty large. Thoughts on creating a new `lib/releaseNotes.ts` file?

Reply:
That's a good idea, I'll merge but an issue for it after summit.